css/styles.css

```diff
@@ -942,7 +935,9 @@ input {
 	margin-left: auto;
 	margin-right: auto;
 }
-
```

Comment:
The code style of the file is to have a blank line between each CSS rule. Please add this back.